component/src/main/ballerina/cassandra/endpoint.bal

```diff
@@ -0,0 +1,230 @@
+};
+
+documentation {
+    Options of the Cassandra native binary protocol.
+
+    F{{sslEnabled}} Enables the use of SSL for the created Cluster
+    F{{noCompact}} Whether or not to include the NO_COMPACT startup option
+    F{{maxSchemaAgreementWaitSeconds}} The maximum time to wait for schema agreement before returning from a DDL query
+    F{{initialProtocolVersion}} Version of the native protocol supported by the driver
+    F{{compression}} Compression supported by the Cassandra binary protocol
+}
+public type ProtocolOptionsConfiguration {
+    boolean sslEnabled,
+    boolean noCompact,
+
+    int maxSchemaAgreementWaitSeconds = -1,
+
+    string initialProtocolVersion,
+    string compression,
+};
+
+documentation {
+    Options related to defaults for individual queries.
+
+    F{{consistencyLevel}} Determines how many nodes in the replica must respond for the coordinator node to
+    successfully process a non-lightweight transaction. Supported values are ANY, ONE, TWO, THREE, QUORUM,
+    ALL, LOCAL_QUORUM, EACH_QUORUM, SERIAL, LOCAL_SERIAL, LOCAL_ONE
+    F{{serialConsistencyLevel}} The serial consistency level is only used by conditional updates
+    (INSERT, UPDATE or DELETE statements with an IF condition). For those, the serial consistency level
+    defines the consistency level of the serial phase (or 'paxos' phase) while the normal consistency level
+     defines the consistency for the 'learn' phase. Supported values are SERIAL, LOCAL_SERIAL
+    F{{defaultIdempotence}} A statement is idempotent if it can be applied multiple times without changing the
+    result beyond the initial application    F{{metadataEnabled}} Toggles client-side token and schema metadata
+    enablement
+    F{{reprepareOnUp}} Determines whether the driver should re-prepare all cached prepared statements on a host
+    when it marks it back up
+    F{{prepareOnAllHosts}} Determines whether the driver should prepare statements on all hosts in the cluster
+    F{{fetchSize}} Sets the default fetch size to use for SELECT queries
+    F{{maxPendingRefreshNodeListRequests}} Determines the maximum number of node list refresh requests that
+    the control connection can accumulate before executing them
+    F{{maxPendingRefreshNodeRequests}} Determines the maximum number of node refresh requests that the control
+    connection can accumulate before executing them
+    F{{maxPendingRefreshSchemaRequests}} Determines the maximum number of schema refresh requests that the
+    control connection can accumulate before executing them
+    F{{refreshNodeListIntervalMillis}} Determines the default window size in milliseconds used to debounce node
+    list refresh requests
+    F{{refreshNodeIntervalMillis}} Determines the default window size in milliseconds used to debounce node
+    refresh requests
+    F{{refreshSchemaIntervalMillis}} Determines the default window size in milliseconds used to debounce node
+    list refresh requests
```

Comment:
Is this description same as refreshNodeListIntervalMillis

Reply:
Need to add the word "schema" in addition. Will fix.